shell_functions.c

```diff
@@ -0,0 +1,264 @@
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h> // for getting the pid
+
+#define ENABLE_DEBUG (1)
+#include "debug.h"
+#include <inttypes.h>
+
+#include "routingtable.h"
+
+#include "eui64.h"
+#include "net/gnrc/ipv6/nib.h"
+
+#include "net/gnrc/ipv6/nib.h"
+#include "net/gnrc/ipv6/nib/nc.h"
+
+#define DUMPER_QUEUE_SIZE (16)
+// constants from the AODVv2 Draft, version 03
+#define DISCOVERY_ATTEMPTS_MAX (1) //(3)
+#define RREQ_WAIT_TIME (2000000) 
+char _rcv_stack_buf[THREAD_STACKSIZE_MAIN];
+
+#define NBR_MAC                                                                \
+  { 0x57, 0x44, 0x33, 0x22, 0x11, 0x00, }
+#define NBR_LINK_LOCAL                                                         \
```

Comment:
Explain usage

Reply:
was just for testing

Reply:
when I was trying to get and save neighbors, it can be removed